tests/layer_tests/conftest.py

```diff
@@ -89,6 +94,12 @@ def use_new_frontend(request):
     return request.config.getoption('use_new_frontend')
 
 
```

Comment:
@dkrupnov, can you also please correct files:
https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/common/layer_test_class.py#L57
https://github.com/openvinotoolkit/openvino/blob/master/.ci/azure/linux.yml#L509 (add `use_legacy_frontend`)
https://github.com/openvinotoolkit/openvino/blob/master/.ci/azure/linux.yml#L516 (add `use_legacy_frontend`)